Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -1360,78 +1369,91 @@ def delete_limit_param(url):
+    try:
+        params = demisto.params()
+        base_url = params['url'].rstrip('/')
+
+        api_token = params.get("credentials", {}).get("password") or params.get('apitoken')
+
+        if not api_token:
+            raise ValueError('Missing API token.')
+
+        verify_certificate = not params.get('insecure', False)
+        proxy = params.get('proxy', False)
+
+        demisto.debug(f'Command being called is {demisto.command()}')
+
+        commands = {
+            'test-module': module_test,
+            'okta-unlock-user': unlock_user_command,
+            'okta-deactivate-user': deactivate_user_command,
+            'okta-activate-user': activate_user_command,
+            'okta-suspend-user': suspend_user_command,
+            'okta-unsuspend-user': unsuspend_user_command,
+            'okta-reset-factor': reset_factor_command,
+            'okta-set-password': set_password_command,
+            'okta-expire-password': expire_password_command,
+            'okta-add-to-group': add_user_to_group_command,
+            'okta-remove-from-group': remove_from_group_command,
+            'okta-get-groups': get_groups_for_user_command,
+            'okta-get-user-factors': get_user_factors_command,
+            'okta-verify-push-factor': verify_push_factor_command,
+            'okta-search': search_command,
+            'okta-get-user': get_user_command,
+            'okta-create-user': create_user_command,
+            'okta-update-user': update_user_command,
+            'okta-get-group-members': get_group_members_command,
+            'okta-list-users': list_users_command,
+            'okta-list-groups': list_groups_command,
+            'okta-get-logs': get_logs_command,
+            'okta-get-failed-logins': get_failed_login_command,
+            'okta-get-application-assignments': get_application_assignments_command,
+            'okta-get-group-assignments': get_group_assignments_command,
+            'okta-get-application-authentication': get_application_authentication_command,
+            'okta-delete-user': delete_user_command,
+            'okta-clear-user-sessions': clear_user_sessions_command,
+            'okta-list-zones': list_zones_command,
+            'okta-get-zone': get_zone_command,
+            'okta-update-zone': update_zone_command,
+            'okta-create-zone': create_zone_command,
+            'okta-create-group': create_group_command,
+            'okta-assign-group-to-app': assign_group_to_app_command,
+            'okta-auth-reset': reset_auth_command,
+        }
 
-    }
+        command = demisto.command()
+        auth_type = AuthType.OAUTH if argToBoolean(params.get('use_oauth', False)) else AuthType.API_TOKEN
+
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            headers={
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+            },
```

Comment:
AS in Packs/ApiModules/Scripts/OktaApiModule/OktaApiModule.py#L233 you send the `auth_headers` - this headers will be ignored
see Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L8887